packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -140,7 +140,7 @@ test.serial('stakeAtom - smart wallet', async t => {
     invitationSpec: {
       source: 'agoricContract',
       instancePath: ['stakeAtom'],
-      callPipe: [['makeCreateAccountInvitation']],
+      callPipe: [['makeAccountInvitation']],
```

Comment:
If you hate `makeMakeAccountInvitation`, how about `makeNewAccountInvitation`?

I'm okey if you don't want to change this.

Reply:
yeah, `makeAccountInvitation` seems misleading

Reply:
oh is it a function to make the maker?